rpmio/macro.cc

```diff
@@ -1774,13 +1781,14 @@ static void pushMacroAny(rpmMacroContext mc,
 }
 
 static void pushMacro(rpmMacroContext mc,
-	const char * n, const char * o, const char * b, int level, int flags)
+		const string & n, const char * o, const string & b,
```

Comment:
I might be missing something, but using `string` on some places while `std::string` on other places seems inconsistent to me.

Reply:
It is, I noticed that too while doing this PR.

There's a *kind of* an inner logic in there: the functions that have a prototype in the header are using std::string in both places, other places take a the string shortcut. We should figure out a standard what to do with these and stick with that, but there are just sooooooo many such decisions to make, at a time when we're not really ready to make them :sweat_smile:

Reply:
In my C++ days, I was never fond of `using`, because then you face this kind of dilemma.

Reply:
Yup. Typing the namespace everywhere seems tedious, but the alternative starts quickly looking worse.